views/card.js

```diff
@@ -23,10 +23,11 @@ module.exports = function gatheringCard (opts) {
 
     const { title, description, startDateTime, image } = state
     const background = image && image.link ? `url('${blobUrl(imageString(image))}')` : ''
-    var date
+    var date, humanReadableDiff
     if (startDateTime && startDateTime.epoch) {
       const t = spacetime(startDateTime.epoch)
       date = `${t.format('date')} ${t.format('month-short')}`
+      humanReadableDiff = spacetime.now().since(t).rounded
```

Comment:
I'm not sure about putting it on the card. I think it's a good idea only if the meeting is soon (like within 24 hours?)
There's a minor challenge where this human readable time will become incorrect over time. This is fine if the card is only open for a bit but a problem if the person leave the page open.

What do you think?

Reply:
What about just move your code into the title... or into an on-hover (ev-hover in mutant) - then we just calculate it at the time when the person interacts to see more info..?

here's what it looks like as a `title` attribute
![image](https://user-images.githubusercontent.com/2665886/49787197-f440c600-fd8a-11e8-925f-8be255460bec.png)

Reply:
To be honest, I didn't think about the case when it gets incorrect over time. That is a very nice point and thus it makes sense to move it to hover. I will test it and update this merge request.

Reply:
@mixmix Just tried to get this working and am hitting a wall. Cannot find any documentation for `ev-hover` seems like its not there in the repository as well (https://github.com/mmckegg/mutant/search?q=ev-hover&unscoped_q=ev-hover). Maybe its called as something else.
Just realized that the issue with title is also similar as title is rendered in the html on load.

Reply:
oh yeah tricky ... mutant needs more docs. Maybe you could make a PR to help out. I made some notes on SSB and mentioned you. Think I got a working demo : %qjeIMZDPjv8Ir9JwOZ7Sy07B1UitB2Bhaz/c9HllpbU=.sha256

I only know this because I've been in this hole before and read through some of the code and talked to matt (i.e. don't feel bad it's hard, it is hard)

Reply:
The notes helped and i've adapted the code similarly. 
At least the `ev-{Event}` part should be added to the docs.